src/utils/userhelper.js

```diff
@@ -0,0 +1,29 @@
+const Bcrypt = require('bcrypt');
+class GameFunction{
+
+    /**
+     * this function use where first time register user  
+     * @param {req} req 
+     * @param {res} res 
+     * @returns account detail
+     */
+    gameFunction(req,res) {
```

Comment:
rename it to `buildUserData()` or something else which convey the purpose of function